Dockerfile

```diff
@@ -108,6 +108,6 @@ RUN chown -R spark_user:spark ${JUPYTERHUB_SECRETS_DIR}
 RUN echo "spark_user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
 # Switch back to non-root user
-USER spark_user
+#USER spark_user
```

Comment:
I keep encountering various permission issues when using the spark_user account. Even after running chown and granting the necessary permissions as root, the problems reappear once I switch back to spark_user.